pkg/volume/csi/csi_plugin.go

```diff
@@ -846,6 +852,46 @@ func (p *csiPlugin) getVolumeLifecycleMode(spec *volume.Spec) (storage.VolumeLif
 	return storage.VolumeLifecyclePersistent, nil
 }
 
+// getFSGroupPolicy returns if the CSI driver supports a volume in the given mode.
+// An error indicates that it isn't supported and explains why.
+func (p *csiPlugin) getFSGroupPolicy(driver string) (storage.FSGroupPolicy, error) {
```

Comment:
Would be nice to have unit tests for this function (in a followup)

Reply:
Addressed in https://github.com/kubernetes/kubernetes/pull/93195